README.md

```diff
@@ -234,9 +234,11 @@ possible security implications.
 
 ### Memory Features
 
-| Feature name   | Description                                                                         |
-| :------------: | :---------------------------------------------------------------------------------: |
-| numa           | Multiple memory nodes i.e. NUMA architecture detected
+| Feature | Attribute  | Description                                                                        |
+| ------- | ---------- | ---------------------------------------------------------------------------------- |
```

Comment:
Please align the header columns with the data columns below, so that it "renders" nicely in a text editor, too

Reply:
Done.